### ISSUE TITLE
Build script does not detect Arch-based distro (Manjaro Linux)

### ISSUE THREAD

**Issue by oxcabe** [flightlessmango/MangoHud#113]
This is my `/etc/os-release` file:

```
NAME="Manjaro Linux"
ID=manjaro
ID_LIKE=arch
PRETTY_NAME="Manjaro Linux"
ANSI_COLOR="1;32"
HOME_URL="https://www.manjaro.org/"
SUPPORT_URL="https://www.manjaro.org/"
BUG_REPORT_URL="https://bugs.manjaro.org/"
LOGO=manjarolinux
```
The script checks for the `NAME` field. I believe it's better to use the `ID_LIKE` field in order to target Arch, Debian, ... based distros. Check [this gist](https://gist.github.com/natefoo/814c5bf936922dad97ff) as well, it contains examples of many `os-release` files for different distros.

I'm also willing to submit a PR for this, shouldn't take too much time to make it work 😉.

**Comment by flightlessmango** [comment]
This should fix it 0cb3bbef502982af5707f368ffe2aabe7c3860f5
I would however recommend using AUR for arch/manjaro

**Comment by oxcabe** [comment]
Thank you!
